bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/Ferroamp.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="energyhub">
+		<label>Ferroamp EnergyHub</label>
+		<description>Bindings for the Ferroamp EnergyHub</description>
```

Comment:
This is a Thing type description, so "Bindings for" seems wrong here. This is only a suggestion for change:
```suggestion
		<description>Represents the Ferroamp EnergyHub</description>
```